### PR TITLE
docs(getting started) : Remove unnecessary diffs

### PR DESCRIPTION
**Pull request by k4a-dev**
There is no diff on the line marked as diff.
So remove the diff and change it to a normal line.

### CODE REVIEW

File: src/content/guides/getting-started.mdx [hunk 1]
```diff
@@ -287,8 +287,7 @@ Given it's not particularly fun to run a local copy of webpack from the CLI, we
    "description": "",
    "private": true,
    "scripts": {
--    "test": "echo \"Error: no test specified\" && exit 1"
-+    "test": "echo \"Error: no test specified\" && exit 1",
+     "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (chenxsan) [inline src/content/guides/getting-started.mdx:290]: There's a `,` in the end of `"test"` line which can't be ignored.

**Comment by chenxsan** [comment]
There's no need to make those changes, the original one is right if you add that `,`.

**Comment by k4a-dev** [comment]
> There's no changes to the code now if you add that `,`, the original one is actually nothing wrong.

Oh, that's right. I am so sorry.
Close this Pull Request.

**Comment by chenxsan** [comment]
> 

No worries, thanks anyway.

**Pull request closed without merging**